app/schema/questionnaire.py

```diff
@@ -22,6 +22,7 @@ def __init__(self):
         self.submission_page = 'summary'
         self.data_version = None
         self.messages = {}
+        self.navigation = False
```

Comment:
if this is required in the schema then we shouldn't default it here? or it shouldn't be required?